pkg/cloud/services/networking/port_test.go

```diff
@@ -439,7 +438,6 @@ func Test_GetOrCreatePort(t *testing.T) {
 			tt.expect(mockClient.EXPECT())
 			s := Service{
 				client: mockClient,
-				logger: logr.DiscardLogger{},
```

Comment:
And again. Is there some behaviour change in logr which means calling methods on a nil logr doesn't error?